src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.CacheEntry.cs

```diff
@@ -87,7 +87,7 @@ static CompletionItem GetAppropriateAttributeItem(CompletionItem attributeItem,
                     if (attributeItem.DisplayText.TryGetWithoutAttributeSuffix(isCaseSensitive: isCaseSensitive, out var attributeNameWithoutSuffix))
                     {
                         // We don't want to cache this item.
-                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix);
+                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix, CompletionItemFlags.Expanded);
```

Comment:
Most of the code change in this PR (adding flags validation in tests) is actually caused by this, which is fixing a second issue also reported in https://github.com/dotnet/roslyn/issues/41601#issuecomment-585499602